airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/ad_account_analytics.json

```diff
@@ -7,7 +7,7 @@
       "format": "date"
     },
     "AD_ACCOUNT_ID": {
-      "type": ["null", "string"]
+      "type": ["null", "string", "number"]
```

Comment:
@drewrasm can you please change the values here to just `["string"]`, as it stated in the [Pinterest documentation](https://developers.pinterest.com/docs/api/v5/#operation/ad_account/analytics) the value for `ad_account_id` should always be a string, and it is always required.